kano_updater/commands/install.py

```diff
@@ -225,7 +225,10 @@ def install_pip_packages(progress):
 
         success = run_pip_command("install --upgrade {}".format(pkg))
 
-        # TODO: abort the install?
         if not success:
             msg = "Installing the '{}' pip package failed".format(pkg)
             logger.error(msg)
+            if not is_internet():
+                msg="Network is down, aborting PIP install"
+                logger.error(msg)
+                raise IOError(msg)
```

Comment:
Hmm, do we want to end everything - if they don't have internet but have already downloaded the debian packages then these can be installed. Maybe we should skip.

Also, in theory, because we cache the downloads in [commands/download.py](https://github.com/KanoComputing/kano-updater/blob/master/kano_updater/commands/download.py#L133-L157) there should be a way of just installing the cached version without needing internet; I realise that the operation that we do here does require internet but the entire install process shouldn't and if we can find a way of not requiring it here then that would be ideal.